src/main/java/core/basesyntax/service/write/FileWriterImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.service.write;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) {
+        try (BufferedWriter writer = new BufferedWriter(new java.io.FileWriter(filePath))) {
+            writer.write(data);
+        } catch (IOException e) {
```

Comment:
Good job using try-with-resources to ensure that resources are closed properly after the operation is complete.